### DISCUSSION TITLE
Number of rows?

### DISCUSSION THREAD

**Discussion by awlobo** [RafaelGB/obsidian-db-folder#623] [Q&A] [answered]
Hi! I can't seem to find info about this but is there any way to change the number of rows displayed in the table? Right now it's 10 rows per page. Could it be more than this value?

**Comment by RafaelGB** [comment, marked as answer]
Hello @awlobo,
Sure! Check our docu here https://rafaelgb.github.io/obsidian-db-folder/features/Viewing%20data/#pagination


We have a max of 250, but you can modify it inside the markdown file:  `pagination_size: 10`

**Reply by awlobo** [reply]
Great! I completely missed it, thanks a lot!

**Comment by PSG2017** [comment]
Hi there. I can't find this number of rows setting. The documentation link above doesn't work - takes me to a weirdly formatted page. I have looked in the database settings but I can't find an item that allows to increase the number of rows displayed per page. If anyone can point me in the right direction I'd be very grateful.

Edit: On my version of DB Folder I am not seeing a row at the top containing next/last arrows and a hamburger menu, as appears on the documentation video relating to rows, and I can't find where to go to find the option makes that row of controls appear.

And I can't find the markdown file!
